src/extensions/default/BrambleUrlCodeHints/camera/interface.js

```diff
@@ -101,6 +117,98 @@ define(function (require, exports, module) {
         // Update the photo component with the snapped photo
         this.photo.update();
 
+        // pattern from http://codepen.io/SitePoint/full/LVpNjp/
+        // $(function() {
+
+          //var Caman = require('caman').Caman;
+          var canvas = document.getElementById("canvas");  // canvas reference here
+          var ctx = canvas.getContext('2d');
+
+          /* Enable Cross Origin Image Editing */
+          var img = new Image();
+          img.crossOrigin = '';
+          img.src = self.photo.data;
+          console.log("img width: "+ img.width + " height " + img.height + "source: " + img.src);
+          console.log("save path in camera is: " + self.camera.savePath);
+
+          img.onload = function() {
+            canvas.width = img.width;
+            canvas.height = img.height;
+            console.log("loaded width: " + img.width + " height " + img.height);
+            ctx.drawImage(img, 0, 0, img.width, img.height);
+          }; // added missing semi-colon
+
+          var filtered = false;
```

Comment:
Instead of differentiating between filtered and non-filtered modes, let's see if we can always filter, so that you can have a `None` filter.  It will make the code cleaner.  Does this work?

```js
Caman('#canvas', img, function() {
    this.render();
});
```

If so, we can just have it render without altering any settings.